playwright/src/main/java/com/microsoft/playwright/impl/Serialization.java

```diff
@@ -215,7 +215,7 @@ public JsonArray serialize(List<KeyboardModifier> modifiers, Type typeOfSrc, Jso
   static JsonArray toJsonArray(Path[] files) {
     JsonArray jsonFiles = new JsonArray();
     for (Path p : files) {
-      jsonFiles.add(p.toString());
+      jsonFiles.add(p.toAbsolutePath().toString());
```

Comment:
Will this remove/resolve `..`? `/a/../foo`

Reply:
It will properly concatenate them with absolute path of cwd, so should be fine.